ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -224,48 +214,47 @@ objects:
             NOW="$(date +%s)"
           done
           [[ "${NOW}" -ge "${TARGET}" ]] && echo "timeout waiting for ${ROUTER_NAMESPACE}/${ROUTER_DEPLOYMENT} to be available" && exit 1
+
+          # wait until the image registry changes propogate to the apiserver to avoid
+          # unnecessary restarts
+          until oc get is -n openshift php 2>/dev/null; do
+            sleep 10
+          done
+          until [[ -n "$( oc get is -n openshift php --template '{{ .status.dockerImageRepository }}' 2>/dev/null )" ]]; do
+            sleep 10
+          done
+          # oh god the blood
+          sleep 180
```

Comment:
This isn't part of router or registry checks ;).  You only need it for e2e, and maybe we'll have proper waiting with `oc` before we're ready to re-enable libvirt e2e.

Reply:
it does help with the waiting for the kube-apiserver to calm down after the kubeadmin user config change, I can remove it though, since there's an until with the kubeadmin login.